zhaquirks/mli/tintE14rgbcct.py

```diff
@@ -0,0 +1,115 @@
+"""Tint E14 RGB CCT."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0x100F  # decimal = 4111
+
+
+class TintRGBCCTColorCluster(CustomCluster, Color):
+    """Tint RGB+CCT Lighting custom cluster."""
+
+    # Set correct capabilities to ct, xy, hs
+    # Tint bulbs do not correctly report this attribute
+    _CONSTANT_ATTRIBUTES = {0x400A: 0b11111}
+
+
+class TintRGBCCTLight(CustomDevice):
+    """Tint E14 RGB+CCT Lighting device."""
+
+    signature = {
+        MODELS_INFO: [("MLI", "tint-ExtendedColor")],
+        ENDPOINTS: {
+            1: {
+                #   "profile_id": 260,
+                #   "device_type": "0x010d",
+                #   "in_clusters": [
+                #     "0x0000",
+                #     "0x0003",
+                #     "0x0004",
+                #     "0x0005",
+                #     "0x0006",
+                #     "0x0008",
+                #     "0x0300",
+                #     "0x1000",
+                #     "0x100f"
+                #   ],
+                #   "out_clusters": [
+                #     "0x0019"
+                #   ]
```

Comment:
Also – this is not wrong – but I think these redundant comments can just be removed, as they don't add any new information.